(dp1
S'output'
p2
S"<type 'exceptions.IOError'> [Errno 2] No such file or directory: 'applications/Mutate/models/testinggithubapi/clonedrepos/68projectdescriptors.txt'"
p3
sS'layer'
p4
S'C:\\Users\\Megan\\Documents\\web2py_win\\web2py\\applications\\Mutate\\controllers/projectmutation.py'
p5
sS'code'
p6
S'__author__ = \'Megan\'\n\nimport glob\nimport os\nimport ast\n\nCLONED_REPOS_PATH = "applications/Mutate/models/testinggithubapi/clonedrepos"\n\n\n@auth.requires_login(otherwise=URL(\'default\',\'login.html\'))\ndef results():\n    task_id=request.args(0)\n    task=db(db.scheduler_task.id==int(task_id)).select().first()\n    no_of_projects = task.vars.split(",")[5].split(":")[1]\n\n    current_number_of_projects_downloaded = 0\n    for files in os.walk(CLONED_REPOS_PATH+"/"+str(task_id)):\n        current_number_of_projects_downloaded = len(map(int, files[1]))\n        print current_number_of_projects_downloaded\n        break\n\n    if int(no_of_projects) == current_number_of_projects_downloaded:\n        if task.status != "COMPLETED":\n            current_number_of_projects_downloaded -= 1\n    else:\n        current_number_of_projects_downloaded -= 1\n    if current_number_of_projects_downloaded < 1:\n        current_number_of_projects_downloaded =0\n\n    for i in range(current_number_of_projects_downloaded):\n        pass\n\n    f = open(CLONED_REPOS_PATH+"/"+str(task_id)+\'projectdescriptors.txt\', "r")\n    temp = f.read()\n    f.close()\n\n    results = H1(\'Found \'+str(current_number_of_projects_downloaded)+\' of \' + str(no_of_projects) + \' projects.\')\n    return dict(results=results, temp=temp)\n\ndef currentsearches():\n    return dict(dict=dict)\nresponse._vars=response._caller(results)\n'
p7
sS'snapshot'
p8
(dp9
sS'traceback'
p10
S'Traceback (most recent call last):\n  File "gluon/restricted.py", line 224, in restricted\n  File "C:\\Users\\Megan\\Documents\\web2py_win\\web2py\\applications\\Mutate\\controllers/projectmutation.py", line 42, in <module>\n  File "gluon/globals.py", line 392, in <lambda>\n  File "gluon/tools.py", line 3439, in f\n  File "C:\\Users\\Megan\\Documents\\web2py_win\\web2py\\applications\\Mutate\\controllers/projectmutation.py", line 33, in results\n    f = open(CLONED_REPOS_PATH+"/"+str(task_id)+\'projectdescriptors.txt\', "r")\nIOError: [Errno 2] No such file or directory: \'applications/Mutate/models/testinggithubapi/clonedrepos/68projectdescriptors.txt\'\n'
p11
s.